### ISSUE TITLE
PHP-error7warning: Declaration of syntax_plugin_searchtablejs::handle should be compatible with DokuWiki_Syntax_Plugin::handle

### ISSUE THREAD

**Issue by tmo26** [xdreamer/searchtablejs#15]
PHP-Version: 7.0.27-1~dotdeb+8.1
Dokuwiki version: 2017-02-19e "Frusterick Manners"
searchtablejs version: 2015-03-18
```
2018/04/05 11:41:07 [error] 704#704: *13136013 FastCGI sent in stderr: "PHP message: PHP Warning:  Declaration of syntax_plugin_searchtablejs::handle($match, $state, $pos, &$handler) should be compatible with DokuWiki_Syntax_Plugin::handle($match, $state, $pos, Doku_Handler $handler) in /var/www/dokuwiki/lib/pluginssearchtablejs/syntax.php on line 0
PHP message: PHP Warning:  Declaration of syntax_plugin_searchtablejs::render($mode, &$renderer, $data) should be compatible with DokuWiki_Syntax_Plugin::render($format, Doku_Renderer $renderer, $data) in /var/www/dokuwiki/lib/plugins/searchtablejs/syntax.php on line 0
```

Since this error is spamming the logs: Can this please be fixed?
Should you need more information, just let me know.

Thanks in advance!

**Comment by tmo26** [comment]
In the meantime, I found the solution:

Edit `/var/www/dokuwiki/lib/plugins/searchtablejs/syntax.php`
  - replace `handle($match, $state, $pos, &$handler)` by `handle($match, $state, $pos, Doku_Handler $handler)`
  - replace `render($mode, &$renderer, $data)` by `render($format, Doku_Renderer $renderer, $data)`
  - replace variable `$mode` with `$format` in the function's body

-> fixed, no warnings any more

**Comment by dregad** [comment]
Fixed by pull request #12
